js/search.js

```diff
@@ -36,7 +48,30 @@ function(ytData, playlist, template, $) {
     }
 
     var displaySearchResults = function(){
-        $("#search_res").empty()
+        $("#search_res").empty();
+        $("#search_res").append("<li><h6 class='ti'>Playlist</h6></li>");
+        if(searchPlaylist===null){
+            $("#search_res").append("<p>Nothing found</p>");
+        }else{
+            for(song in searchPlaylist){
+                var vidrow=template.getVidRow(searchPlaylist[song]);
+                $(vidrow).addClass("item-search");
+                $(vidrow).on("click",function(){
+                    //console.log('clicked '+this.id);
+                    var hashId="#playlist > #"+this.id;
+                    $("#playlist").animate({
+                            scrollTop : $(hashId).offset().top
+                        }, 2000);
```

Comment:
This scroll doesn't work too well. 

![search_playlist](https://user-images.githubusercontent.com/7384411/33899011-2aac4d6e-df90-11e7-972d-f20d50e2f24c.png)

Maybe you have to subtract the `<ul>`'s top from the element's top to get a proper scroll value.